bangazon/bangazon_ui/models/order_model.py

```diff
@@ -0,0 +1,25 @@
+import sys
+sys.path.append("../")
+from django.db import models
+from django.contrib.auth.models import User
+from bangazon_ui.models.customer_model import Customer
+from bangazon_ui.models.payment_type_model import PaymentType
+from bangazon_ui.models.product_model import Product
+
+
+
+class Order (models.Model):
+    """
+    The Order table pulls information from PaymentType and Product via a join 
+    """
+
+    customer = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Thanks for that correction @Onodric  / @juliakimchung  checkout this doc for the Meta class line of code in the 'order_model.py' file . http://stackoverflow.com/questions/677172/verbose-name-plural-unexpected-in-a-model